src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,61 @@
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  itemsLeft: Todo[],
+  filterType: string,
+  handleSelectedType: (event:React.MouseEvent<HTMLAnchorElement>) => void,
+};
+
+export const Footer: React.FC<Props> = ({
+  itemsLeft,
+  filterType,
+  handleSelectedType,
+}) => {
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${itemsLeft.length} items left`}
+      </span>
+
+      <nav className="filter">
+        <a
+          href="#/"
+          className={classNames('filter__link', {
+            selected: filterType === 'all',
+          })}
+          type="all"
+          onClick={handleSelectedType}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={classNames('filter__link', {
+            selected: filterType === 'active',
+          })}
+          type="active"
```

Comment:
The 'type' attribute is not valid for <a> element. To pass the filter type to the onClick handler, you can use a closure function.